blocks_vertical/data.js

```diff
@@ -527,6 +527,7 @@ Blockly.Constants.Data.CUSTOM_CONTEXT_MENU_GET_VARIABLE_MIXIN = {
       for (var i = 0; i < variablesList.length; i++) {
         var varName = variablesList[i].name;
         if (varName == currentVarName) continue;
+
```

Comment:
Nit: stray newline

```suggestion
```

Reply:
This is so that it's consistent with the `CUSTOM_CONTEXT_MENU_GET_VARIABLE_MIXIN` function: https://github.com/towerofnix/scratch-blocks/blob/564d3ca8c22aa2c0c6f240ab3369b4093b2a1934/blocks_vertical/data.js#L527-L532

I can commit your suggestion if you want, but I figure code consistency is nice and it'll likely be a while (extensionification) before this code is touched again :P

No worries about the wait time! It was set in the May milestone after all :P

Reply:
Ah, I assume you mean so that it's consistent with the `..._LIST_MIXIN` function.

That's a good point, we can keep the newline. Thanks!